xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1672,7 +1673,7 @@ namespace VIDEO
     CFileItemList availableArtFiles;
     CDirectory::GetDirectory(path, availableArtFiles,
         CServiceBroker::GetFileExtensionProvider().GetPictureExtensions(),
-        DIR_FLAG_NO_FILE_DIRS | DIR_FLAG_READ_CACHE | DIR_FLAG_NO_FILE_INFO);
```

Comment:
I changed this as I found that if you add a file to the MSIF once Kodi has started it never gets found on subsequent searches. GIven the MSIF will only contain a small number of files I would have this doesn't have a performance impact.